server/Pipfile

```diff
@@ -13,6 +13,9 @@ passlib = "*"
 flask-sqlalchemy = "*"
 psycopg2-binary = "*"
 python-dotenv = "*"
+pylint-flask = "*"
+redis = "*"
+rq = "*"
```

Comment:
Can we start writing specific version numbers (in case we run into version issues in the future)